app/controllers/spina/registers_controller.rb

```diff
@@ -19,8 +21,21 @@ def index
       @current_phases = Spina::Register::CURRENT_PHASES
     end
 
+    def new_show
+      @register = Spina::Register.find_by_slug!(params[:id])
+
+      @register_data = @@registers_client.get_register(@register.name.parameterize, @register.register_phase)
```

Comment:
This doesn't need to be an instance variable, could just be defined inside of the method

Reply:
I am actually using this instance variable in the view so iv'e kept it 👍